recipes/missingno/meta.yaml

```diff
@@ -0,0 +1,38 @@
+{% set name = "missingno" %}
+{% set version = "0.3.5" %}
+{% set sha256 = "385c7bee14b6ce45c86266e5428544ac9e1cad9c3fe55048fe84e5b552308b17" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  script: python setup.py install --single-version-externally-managed --record record.txt
+
+requirements:
+  build:
+    - python
+    - setuptools
+  run:
+    - python
```

Comment:
My bad, boils down to bad memory about what needs to be done. Thanks for double-checking!